consensus/service/src/byzantine_ledger/worker.rs

```diff
@@ -366,19 +362,12 @@ impl<
                 // SCP Statement
                 TaskMessage::ConsensusMsg(consensus_msg, from_responder_id) => {
                     // Only look at messages that are not for past slots.
-                    if consensus_msg.scp_msg().slot_index >= self.cur_slot {
-                        // Feed network state. The sync service needs this
-                        // to be able to tell if we fell behind based on the slot values.
-                        // Block ID checking is skipped here, since if we fell behind
-                        // we are not going to have blocks to compare to.
+                    if consensus_msg.scp_msg().slot_index >= self.current_slot_index {
```

Comment:
Wasn't part of the idea of the work done here to allow for nodes to get back to nodes that are a bit behind by properly handling the externalize phase? This would ignore any nodes that are currently behind.

Reply:
Good catch! Yes, the intent is that it should process messages for the current slot and however many recently externalized slots that Node is keeping around. I will update this and run it through another round of testing.